airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -78,6 +101,8 @@ def ti_state_discriminator(v: dict[str, str] | BaseModel) -> str:
         return str(state)
     elif state in set(TerminalTIState):
         return "_terminal_"
+    elif state == "deferred":
```

Comment:
```suggestion
    elif state == TIState.DEFERRED:
```
to be consistent with L100

Reply:
No problem. Made the change